apps/service-portal/src/components/Sidebar/Sidebar.tsx

```diff
@@ -1,45 +1,92 @@
-import React, { FC } from 'react'
-import { Box, Stack, Text } from '@island.is/island-ui/core'
-import * as styles from './Sidebar.css'
+import React, { FC, useState } from 'react'
+import { Box, Stack, Logo, FocusableBox, Icon } from '@island.is/island-ui/core'
+import { BetaTag } from '../Logo/BetaTag'
+import { ActionType } from '../../store/actions'
+import { ServicePortalPath } from '@island.is/service-portal/core'
+import { Link } from 'react-router-dom'
+import { useStore } from '../../store/stateProvider'
 import ModuleNavigation from './ModuleNavigation'
 import useNavigation from '../../hooks/useNavigation/useNavigation'
-import { useLocale } from '@island.is/localization'
-import { ServicePortalPath, m } from '@island.is/service-portal/core'
+import { useUpdateUnreadDocuments } from '@island.is/service-portal/core'
+import * as styles from './Sidebar.css'
+import cn from 'classnames'
 
 export const Sidebar: FC<{}> = () => {
-  const { formatMessage } = useLocale()
   const navigation = useNavigation()
+  const [{ sidebarState }, dispatch] = useStore()
+  const [collapsed, setCollapsed] = useState(sidebarState === 'closed')
+  const badgeContext = useUpdateUnreadDocuments()
 
   return (
-    <aside className={styles.sidebar}>
-      {navigation.map((rootItem, rootIndex) => (
+    <aside className={cn(styles.sidebar, collapsed && styles.collapsed)}>
```

Comment:
Design comment:

I wish the collapsed state could be a bit tighter, there's just soo much horizontal whitespace around the small navigation icons. This is a big waste on tablet sized window sizes but sticks out to me personally on all screen sizes.

![image](https://user-images.githubusercontent.com/115094/146917744-00e59265-dbca-42ee-8d40-8ba5eb7fb2de.png)

I imagine it's "beta"'s fault. Could we collapse the beta below the logo instead?

Reply:
I'll take this to Keli, the designer

Reply:
Also, can we have the consistent whitespace on the left and right side of the main content. Currently it uses `margin-left: 25%;` and `16%` which looks weird to me and wastes space on smaller screens. Maybe we can use some fancy `calc()` to solve this.

![image](https://user-images.githubusercontent.com/115094/146945494-c0e59b59-ca57-49fa-88c7-e00519e51959.png)

Reply:
Yeah I agree, let's find a solution to this 👍

Reply:
Layout issues - The grid was wrong, was wider then it should be on the right side, it looks better for now. We are waiting for xxl breakpoint to be merged and then we will refactor this. 
Collapsed state - should stay like this for now at least

Reply:
Sorry. This still looks weird to me.

<img width="983" alt="Screenshot 2022-01-05 at 21 40 03" src="https://user-images.githubusercontent.com/115094/148293878-a50bc817-75b5-49a2-996a-a72ac4bc3494.png">

There is similar inconsistency in larger screens.

IMHO, our content container should always have the same spacing on left and right (to sidebar and page edge). On smaller screen sizes, we should use all available space except for 24px or whatever whitespace aligns with the overall page margin.

Reply:
It looks like the horizontal whitespace of the page content is now spread over 3 elements. It would be nice if they could be simplified somehow. Happy to jump on a call to discuss the issue or brainstorm css for this.